src/main/java/uk/ac/cam/cl/dtg/segue/api/QuestionFacade.java

```diff
@@ -437,4 +529,13 @@ public Response generateSpecification(@Context final HttpServletRequest request,
       return responseWrapper.toResponse();
     }
   }
+
+  // Move these out of the facade
```

Comment:
This looks like a forgotten TODO.

Reply:
I just noticed what's going on: this is copy/pasted from somewhere else. So yes, the comment is right, this (and the original) needs to be moved somewhere else.